modules/ztp-sno-siteconfig-config-reference.adoc

```diff
@@ -80,18 +80,12 @@ When creating the `bmh-secret` CR, use the same namespace as the `SiteConfig` CR
 The default value is `UEFI`. Use `UEFISecureBoot` to enable secure boot on the host.
 
 |`spec.clusters.nodes.rootDeviceHints`
-|Specifies the device for deployment. Identifiers that are stable across reboots are recommended, for example, `wwn: <disk_wwn>` or `deviceName: /dev/disk/by-path/<device_path>`. For a detailed list of stable identifiers, see the "About root device hints section".
-
-|`spec.clusters.nodes.diskPartition`
-|Optional. The provided example `diskPartition` is used to configure additional disk partitions.
+|Specifies the device for deployment. Identifiers that are stable across reboots are recommended, for example, `wwn: <disk_wwn>` or `deviceName: /dev/disk/by-path/<device_path>`. `<by-path>` values are preferred. For a detailed list of stable identifiers, see the "About root device hints" section.
```

Comment:
Run on sentence. Separate into two sentences: "Identifiers that are stable across reboots are recommended. For example, `wwn: <disk_wwn>` or `deviceName: /dev/disk/by-path/<device_path>`. `<by-path>` values are preferred.